### DISCUSSION TITLE
Library/Store Comparison for books in your library but not "owned" in the Store...

### DISCUSSION THREAD

**Discussion by CTinMich** [joonaspaakko/audible-library-extractor#101] [Ideas]
Great App... just found it :)  The reason I was looking for a library export tool was I found a lot of my books in my "Library" show up in the Audible "Store" as not owned (i.e "not in library".  Audible had couple reasons why this might happen (such as same book, but publisher lost rights to new company?).  I found of about 8 titles that were broken like this when I decided to re-listen to one of my old series and I checked to see if there were any new titles in the series.  In the "store" search all my old books showed as "not in my library" and wanted me to re-buy them, but...  If I went to my Library I saw them and had full access.  Audible worked with me to switch my access to the current titles with out issue... just a pain in the butt.  I certainly do not want to "manually" compare the library and store copies if I can help it hahaha  I have over 600+ book in my Library.

So...  any interest on adding a feature to list if a volume in the library does not match the store copy?  This issue may cause people to accidently buy a book they already own, or at least make a nuisance  in re-listing to previously read books.

Thanks for your time and cool app :)

**Comment by joonaspaakko** [comment]
Your Audible library is what tells you if you own a book, not the store. Just like you don't go to a mom & pop book store to confirm if you still have your copy of the book you purchased from there. And if there is a new release of the book, they are under no obligation to give you that for free. Obviously, if you search for a book in the Audible store pages and you don't have whatever the current version(s) are, it will only give you the option to buy it, because you have an older version in your library. And unfortunately, the series pages are directly tied to the store and so you won't find any books there that are no longer sold. Just the library.

The one potentially confusing situation is if you try to click the "book details" button in your Audible library and it says the store page is missing) This is how many people end up believing they no longer own the book, but that it's only the store page that is missing. Whether it's on the app or the website, you can always play or download a book directly from the library page, without having to go to the store page. Another reason why people think the book might be missing from their library is that it doesn't come up in the mobile app search, but that's usually just it being unreliable and you should confirm it by searching from the desktop website library.

So, again, when they stop selling a book, it doesn't disappear from your library, just the store. And while Audible is pretty trigger happy with throwing credits at customers, they are under no obligation to give you the new release for free. I'd honestly be pretty surprised if they kept giving you free credits for entire series, given that you already bought it and still very much own the books, but then again they don't seem to be too into debugging anything, so rather than go through the process of confirming if you can access the books or not they often do seem to give out credits.

- - - - 

I believe there already is a filter for that: ["Store page unavailable"](https://joonaspaakko.github.io/my-audible-library/#/library?filterExtras=store-page-unavailable). This is an example with the filter enabled in my library. You can see it by clicking the funnel icon on the search bar.

### This filter covers 2 similar criteria:

1. **Book store page can no longer be accessed**
    - Book can no longer be purchased _for whatever reason_
    - This doesn't meant there isn't another version out there, just that this particular version is no longer being sold.
 2. **Book store page changed**
     - The book is no longer being sold and its store page was at somepoint redirected to a new store page. 
     - This should be relatively rare.
     - These books are shown under the same filter because we can no longer trust the store page details to be accurate to the version in users library.
     - If you need to see this distinction, for these books with a missing store page there is a type of label in book details under the title saying "missing". That's true most books in the above link, but for [The Martian](https://joonaspaakko.github.io/my-audible-library/#/library?filterExtras=store-page-unavailable&book=B00B5HZGUG) it says "changed" because the old store page was kind of overtaken by the new release.

**Comment by CTinMich** [comment]
Awesome :) thanks!!!

Sent from my Verizon, Samsung Galaxy smartphone
Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
